beginner-challenges/challenge-2/index.js

```diff
@@ -0,0 +1,31 @@
+
+function handleUserInput() {
```

Comment:
I would like to have two additional functions.

The first function takes in a number and returns a two-dimensional array with multiplications. The second function accepts a two-dimensional array and returns the table.